MapboxNavigation/NavigationMapView.swift

```diff
@@ -21,6 +23,10 @@ open class NavigationMapView: MGLMapView {
         22: MGLStyleValue(rawValue: 18)
     ]
     
+    let TrafficSevere = UIColor(red:0.54, green:0.06, blue:0.22, alpha:1.0)
+    let TrafficHeavy = UIColor(red:0.91, green:0.20, blue:0.25, alpha:1.0)
+    let TrafficModerate = UIColor(red:0.95, green:0.65, blue:0.31, alpha:1.0)
```

Comment:
Even if we decide that no one should be allowed to deviate from the orange/red/maroon color scheme, a designer may want to deviate from the exact colors `#F2A64F` `#E83340` `#8A0F38`, for example to better match the Traffic Night style.

For what it’s worth, I don’t think it would be very risky to expose these colors in Style like all the others. If the developer chooses a misleading color scheme of their own, that doesn’t reflect poorly on our Directions API any more than an ugly custom style would reflect poorly on the Mapbox Streets source or the map SDK.